README.md

```diff
@@ -1,131 +1,75 @@
- * [Online Demos](#online-demos)
- * [Documentation](#documentation)
+![Dashboard View](https://github.com/boldbi/aspnet-core-sample/assets/91586758/4af68f49-ffc0-400a-a323-55a3f3600a1d)
 
- ## Requirements to run the demo
-
-The samples require the following requirements to run.
-
- * [Visual Studio 2022](https://visualstudio.microsoft.com/downloads/)
+ ## Requirements/Prerequisites
+ 
  * [.NET Core 6.0](https://dotnet.microsoft.com/en-us/download/dotnet-core)
  * [Node.js](https://nodejs.org/en/)
- * [Visual Studio Code](https://code.visualstudio.com/download)
 
- ## Using the Angular with ASP.NET Core sample
+ > **NOTE:** Node.js versions 14.16 to 18.16 are supported.
+
+ #### Help link
+
+ * https://help.boldbi.com/embedded-bi/faq/where-can-i-find-the-product-version/
+
+ #### Supported browsers
+  
+  * Google Chrome, Microsoft Edge, Mozilla Firefox, and Safari.
+
+ ## Configuration
+
+  * Please ensure that you have enabled embed authentication on the `embed settings` page. If it is not currently enabled, please refer to the following image or detailed [instructions](https://help.boldbi.com/embedded-bi/site-administration/embed-settings/#get-embed-configuration-file) to enable it.
+  ![Embed Settings](https://github.com/boldbi/aspnet-core-sample/assets/91586758/b3a81978-9eb4-42b2-92bb-d1e2735ab007)
+
+  * To download the `embedConfig.json` file, please follow this [link](https://help.boldbi.com/embedded-bi/site-administration/embed-settings/#get-embed-configuration-file) for reference. Additionally, you can refer to the following image for visual guidance.
+    ![Embed Settings Download](https://github.com/boldbi/aspnet-core-sample/assets/91586758/d27d4cfc-6a3e-4c34-975e-f5f22dea6172)
+    ![EmbedConfig Properties](https://github.com/boldbi/aspnet-core-sample/assets/91586758/d6ce925a-0d4c-45d2-817e-24d6d59e0d63)
+
+  * Copy the downloaded embedConfig.json file and paste it into the designated [location](https://github.com/boldbi/react-with-aspnet-core-sample/tree/master/AngularwithASP.NETCore) within the application. Please ensure that you have placed it in the application as shown in the following image.
+    
+    ![EmbedConfig image](https://github.com/boldbi/aspnet-core-sample/assets/91586758/bdb83a3e-02e4-4e99-ad57-717438e5ec5c)
+
+## Run a Sample Using Command Prompt
+
+  1. Open the command prompt and navigate to the specified file [location](https://github.com/boldbi/angular-with-aspnet-core-sample/tree/master/ASP.NET%20Core/BoldBI.Embed.Sample) where the project is located.
+
+  2. Execute the command `dotnet restore` to restore the necessary packages. Once the packages have been successfully restored, use the `dotnet build` command to build the project.
+
+  3.  To install all dependent packages, use the following command `npm install`. To install the Bold BI Embedded SDK package, use the following command `npm install -save @boldbi/boldbi-embedded-sdk`.
```

Comment:
Why still you are having steps to install npm boldbi package. It would be installed if run npm install command. Already I have discussed about this with you.

Reply:
Have you checked about how do we define the embedded-sdk package to get updated version.

Reply:
Remove unwanted files as said in previous PR.

Reply:
Unwanted files are removed. npm install boldbi package removed. 
embedded-sdk package version also how to found are checked. Like below command.

npm install @boldbi/boldbi-embedded-sdk@6.4.6